### PR TITLE
CDS-59027-added inline and Harness file store options in manifest ove…

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
…rride

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md [hunk 1]
```diff
@@ -201,22 +201,29 @@ For more information on command flags, go to [Apply](https://kubernetes.io/docs/
...
+   * Select **Values YAML** type, and select **Continue**.
+     * Select a values YAML store. You can select remotes stores such as Git, GitHub, GitLab, Bitbucket, Azure Repo or Harness file store, or Inline.
+       - If you select a remote store, select or add a connector to that repo, and then enter a path to the folder or file.
+       - If you select Harness file store, select a values YAML file from project, organization, or account in the file store.
+       - If you select **Inline**, enter a name for you to identify values YAML and the `name:value` pairs for the override.
+   * Select **Kuztomize Patches** type, and select **Continue**.
+     * Select a Kuztomize Patches store. You can select remotes stores such as Git, GitHub, GitLab, Bitbucket, Azure Repo or Harness file store, or Inline.
+       - If you select a remote store, select or add a connector to that repo, and then enter a path to the folder or file.
+       - If you select Harness file store, select a Kuztomize Patch file from project, organization, or account in the file store.
+       - If you select **Inline**, enter a name for you to identify the Kuztomize Patch file and the value for the override.
```
Comment (namantalaycha10) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md:225]: we can add this as example of kustomize patch content

apiVersion: apps/v1
kind: Deployment
metadata:
  name: example-deploy
  namespace: default
spec:
  template :
    spec:
      containers:
        - name: example-app
          image: <+serviceVariables.image>
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: example-deploy
  namespace: default
spec:
  replicas: <+serviceVariables.replica>

**Review by namantalaycha10 — DISMISSED** [review]

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md [hunk 2]
```diff
@@ -201,22 +201,55 @@ For more information on command flags, go to [Apply](https://kubernetes.io/docs/
...
-You can also add a values YAML values and/or files in the Apply Step **Override Value**.
-
-1. Click **Add Manifest**.
-2. Select a values YAML type and click **Continue**.
-3. Select a values YAML store. You can select remote or inline.
-   - If you selected a remote store, select or add a connector to that repo, and then enter a path to the folder or file.
-   - If you selected **Inline**, enter a name for you to identify values YAML and the `name:value` pairs for the override.
-4. Click **Submit**. 
+You can also add a values YAML values and/or files or Kuztomize Patch files in the Apply step **Override Value**.
+
```
Comment (SmritiSatya) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md:213]: You can also add YAML values and/or files or Kustomize patch files in the Apply step **Override Value**.

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md [hunk 3]
```diff
@@ -201,22 +201,55 @@ For more information on command flags, go to [Apply](https://kubernetes.io/docs/
...
+1. Select **Add Manifest**.
+2. In **Specify Manifest Type**, select **Values YAML** or **Kuztomize Patches**, and then select **Continue**.
```
Comment (SmritiSatya) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md:215]: Is it Kuztomize or Kustomize? (global)

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md [hunk 4]
```diff
@@ -201,22 +201,55 @@ For more information on command flags, go to [Apply](https://kubernetes.io/docs/
...
+   * To add a values YAML value and/or files, select a values YAML store. You can select remotes stores such as Git, GitHub, GitLab, Bitbucket, Azure Repo or Harness file store, or Inline.
```
Comment (SmritiSatya) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md:216]: To add a YAML value? I am not sure I understand the statement.
Also, "You can select remote stores" or "You can select remotes store", right?
... Azure Repo, Harness file store, or Inline.

**Review by SmritiSatya — APPROVED** [review]

**Review by abosii-harness — APPROVED** [review]

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md (hunk 2 above)
Comment (roshnisarangadharan) [reply docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md:213]: values YAML is a file.

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md (hunk 3 above)
Comment (roshnisarangadharan) [reply docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md:215]: Good catch! It is Kustomize.

File: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md (hunk 4 above)
Comment (roshnisarangadharan) [reply docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-executions/deploy-manifests-using-apply-step.md:216]: Will make the change

**Pull request merged**